actions/check_is_bot_in_voice_channel.js

```diff
@@ -100,7 +100,13 @@ module.exports = {
 	action(cache) {
 		const data = cache.actions[cache.index];
 		const Audio = this.getDBM().Audio;
-		const result = cache.server ? Audio.getSubscription(cache.server) : false;
+		let result = false;
+		if (cache.server) {
+			result = Audio.getSubscription(cache.server);
```

Comment:
Isn't this action supposed to store a boolean value?

```suggestion
			result = !!Audio.getSubscription(cache.server);
```

Reply:
Yes it should, I didn't think about that and kept it the same as Robert had it.